.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8u171-jdk-node-browsers
+    environment:
+      GRADLE_OPTS: -Dorg.gradle.console=plain -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false
+
+    steps:
+      - checkout
+      - restore_cache:
```

Comment:
is there way not copy&paste these 20ish lines for restore&save cache everywhere?

Reply:
Not that i know of from the [doc](https://circleci.com/docs/2.0/caching/),  @jmcampanini?

Reply:
you would need to use yaml anchors, etc. it's pretty useful since you only name the caches in 1 place, but using anchors makes the yaml harder to read in general. tradeoffs until circle implements "orbs" (re-usable steps that are defined by us)

Reply:
anchors are only useful if they are being used multiple times, the save cache or restore cache is used once here.